### PR TITLE
Improve templates for GitHub

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add emoji and a support section
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | No need QA

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/19539)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/3_support_question.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: ⛔ Support Question
```
Comment (eternoendless) [inline .github/ISSUE_TEMPLATE/3_support_question.md]:
```suggestion
name: ⛔ I need help
```

File: .github/ISSUE_TEMPLATE/3_support_question.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+about: See https://www.prestashop.com/support for questions about using PrestaShop
+
+---
+
+We use GitHub issues only to discuss about PrestaShop bugs and new features. For
+this kind of questions about using PrestaShop or third-party modules, please use
+any of the support alternatives shown in https://www.prestashop.com/support
```
Comment (eternoendless) [inline .github/ISSUE_TEMPLATE/3_support_question.md]:
```suggestion
We use GitHub issues only to discuss about bugs in the PrestaShop project and new features.
If you have questions about using PrestaShop or third-party modules, or if you need help
with your shop, please refer to one of our support plans: https://www.prestashop.com/support
```

**Review by matthieu-rolland — APPROVED** [review]

**Review by SimonGrn — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks me (and @eternoendless :heart: )

**Comment by matks** [comment]
😱 euh guys ...
```
If you have questions about using PrestaShop or third-party modules, or if you need help
with your shop, please refer to one of our support plans: https://www.prestashop.com/support
```

So basically "you have a question, buy something" 😅 sounds a bit ... raw ?

May I suggest this:
```
3_support_question.md
---
name: ⛔ I need help
about: For questions about using PrestaShop

---

We use GitHub issues only to discuss about bugs in the PrestaShop project and new features.
If you have questions about using PrestaShop or third-party modules, you can look for help on the Forum or on the Slack channel.
If you need help with your shop, please refer to one of our support plans: https://www.prestashop.com/support

Thanks!
```

Not all questions are good for support 😄 here are some examples:
- is PrestaShop compatible with php7.4 ?
- do you have an SDK for the API ?
- if I want to add some puppeeter tests to validate my PR, how could I do it ?
- is there a list of migrated pages / legacy pages for 1.7.7 ?

**Comment by eternoendless** [comment]
@matks Please open a PR, we can discuss there.

**Comment by matks** [comment]
> @matks Please open a PR, we can discuss there.

Gnagnagna it's such a heavy process gnagnagna it's at least 4 clics gnagnagna I'm wasting my time here fixing your mistakes :trollface:

**Comment by matks** [comment]
> > @matks Please open a PR, we can discuss there.
> 
> Gnagnagna it's such a heavy process gnagnagna it's at least 4 clics gnagnagna I'm wasting my time here fixing your mistakes :trollface:

Joke aside, here's the PR 😊 https://github.com/PrestaShop/PrestaShop/pull/19567

**Pull request merged**